src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -853,11 +963,28 @@ void TableDockWidget::setGroupLabel(char label) {
       QVariant v = item->data(0, Qt::UserRole);
       PeakGroup *group = v.value<PeakGroup *>();
       if (group != NULL) {
+        if (!(group->label=='g'||group->label=='b')){
+          numberOfGroupsMarked+=1;
+          if (numberOfGroupsMarked == 1) curationTimer.start();
+          group->setLabel(label);
+          subsetPeakGroups.push_back(*group);
+        }
         group->setLabel(label);
+        if (numberOfGroupsMarked ==10){
+          if (_mainwindow->sessionCount==1){
+            // This code is used to show the initial inferences from the cloud model.
+            Q_EMIT(tenPeaksMarked(curationTimer.elapsed()));
+          }
+          _mainwindow->sessionCount+=1;          
```

Comment:
sessionCount is not required either. It was used to ensure the "accuracy and time of ML model" toast is only shown once per session.